cli_tools/common/utils/storage/storage_client.go

```diff
@@ -186,6 +187,16 @@ func SplitGCSPath(p string) (string, string, error) {
 	return "", "", fmt.Errorf("%q is not a valid GCS path", p)
 }
 
+// SplitBucketGCSPath splits GCS path to get bucket name
+func SplitBucketGCSPath(p string) (string, error) {
```

Comment:
Rename to GetBucketNameFromGCSPath(gcsPath string) or similar current name is not appropriate for what this func does.